pennylane/ops/channel.py

```diff
@@ -223,6 +223,90 @@ def _kraus_matrices(cls, *params):
         return [K0, K1, K2, K3]
 
 
+class BitFlipChannel(Channel):
```

Comment:
I actually like dropping the "Channel" and naming these `BitFlip` and `PhaseFlip`. The only case we had "Channel" in the name I believe was for `DepolarizingChannel`, which makes sense since just `Depolarizing` is a bit akward

Reply:
:100:

Reply:
Fair enough! How do you feel about `BitFlipError` and `PhaseFlipError`? I'd like to have something to denote that they are error channels and not just regular operations (also I just noticed I that this is what I called them in the changelog... :facepalm:)

Reply:
I have this discussion with Alain all the time...

I personally prefer names to be implicit and short, rather than explicit and long. So while you're right that a "BitFlip" could potentially be interpreted as a regular operation (apply PauliX), it will likely be clear from context and from a good docstring that this is not the case. The more compact name then leads to cleaner and shorter code.